manifest_staging/charts/csi-secrets-store-provider-azure/arc-values.yaml

```
@@ -193,3 +193,6 @@ Azure:
 # Arc monitoring
 arc:
   enableMonitoring: true
+
+# Port that serves metrics
+providerMetricsAddr: "8898"
```

```suggestion
metricsAddr: "8898"
```

This value is only for the provider chart and doesn't conflict with driver the `provider` prefix in name isn't required